sig-multicluster/README.md

```diff
@@ -36,15 +36,15 @@ The Chairs of the SIG run operations and processes governing the SIG.
 ## Subprojects
 
 The following subprojects are owned by sig-multicluster:
+- **Kubefed**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes-sigs/Kubefed/master/OWNERS
```

Comment:
This needs to be generated by running `make`. 

The test fails because it should be s/Kubefed/kubefed here.

Reply:
Weird, i'm not sure this is the right version that i pushed...